src/lib/y2network/sysconfig/connection_config_writer.rb

```diff
@@ -42,7 +42,6 @@ def write(conn, old_conn = nil)
         handler_class = find_handler_class(conn.type)
         return nil if handler_class.nil?
 
-        remove(old_conn) if old_conn
```

Comment:
Please, check the `remove` method as it seems to do more than just removing a file. Perhaps you might only need to modify that method instead of removing this line.

Reply:
After some discussion we agreed that this removal is not needed as we are caching all the SCR values and the unmanaged will be written again correctly.

If we do not remove the interface then we need to take into account the interface renaming case.